### DISCUSSION TITLE
Disable one of the analog ROI?

### DISCUSSION THREAD

**Discussion by johanneseckert** [jomjol/AI-on-the-edge-device#1687] [Ideas] [answered]
On  my water meter the third analog counter (0.01x) covered by a logo on the casing.

is there a way to disable/skip/assume it? Or can I only use the first two analog counters in my case?

<img width="910" alt="Screenshot 2022-12-26 at 5 49 36 PM" src="https://user-images.githubusercontent.com/545926/209569393-c9896bd8-4c57-4ef4-8fc2-4812a18989aa.png">

**Comment by johanneseckert** [comment]
An idea: for increased precision with a limited number of analog gauges, could there be a setting to use the decimal point from the ML model analysis?

![Screenshot_20221226-201100](https://user-images.githubusercontent.com/545926/209577834-de96c06e-bad6-4307-b6e9-372e8dfb250e.png)

For this example:
Clear reading: 127.62
Precision using ML model: 127.624

**Comment by caco3** [comment, marked as answer]
I think you must do following:
 1. remove the 4th ROI (0.0001m3)
 1. enable `ExtendedResolution` in the settings

This way you should get some interpolation for the 3th ROI based on the 2nd ROI.

How ever I don't think you can use the 4th ROI at the same time.

**Reply by johanneseckert** [reply]
whoop!
Thank you, that was straightforward, and I could have read through all of the options to find it. thank you
